### ISSUE TITLE
When trying to install devika, the app crashes

### ISSUE THREAD

**Issue by stefanp2021** [oven-sh/bun#11120]
### How can we reproduce the crash?

https://github.com/stitionai/devika?tab=readme-ov-file#installation
 C:\Users\stefan.pirker\Documents\MEGA\Coding\devika\ui> bun install
bun install v1.1.8 (89d25807)
  🔒 Saving lockfile... ============================================================
Bun v1.1.8 (89d25807) Windows x64
Args: "C:\Users\stefan.pirker\.bun\bin\bun.exe", "install"
Elapsed: 210ms | User: 0ms | Sys: 15ms
RSS: 51.59MB | Peak: 51.59MB | Commit: 92.21MB | Faults: 12779

panic(main thread): Internal assertion failure
oh no: Bun has crashed. This indicates a bug in Bun, not your code.

### Relevant log output

```shell
C:\Users\stefan.pirker\Documents\MEGA\Coding\devika\ui> bun install
bun install v1.1.8 (89d25807)
  🔒 Saving lockfile... ============================================================
Bun v1.1.8 (89d25807) Windows x64
Args: "C:\Users\stefan.pirker\.bun\bin\bun.exe", "install"
Elapsed: 210ms | User: 0ms | Sys: 15ms
RSS: 51.59MB | Peak: 51.59MB | Commit: 92.21MB | Faults: 12779

panic(main thread): Internal assertion failure
oh no: Bun has crashed. This indicates a bug in Bun, not your code.
```

### Stack Trace (bun.report)

Bun v1.1.8 ([`89d2580`](https://github.com/oven-sh/bun/tree/89d25807fbf4488d3bfcb116484d0c2cf824f98c)) on windows x86_64 [InstallCommand]

**panic**: Internal assertion failure

- [`debug.zig:373`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/deps/zig/lib/std/debug.zig#L373): `panicExtra`
- [`debug.zig:348`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/deps/zig/lib/std/debug.zig#L348): `panic`
- [`output.zig:356`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/output.zig#L356): `panic`
- [`bun.zig:3177`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/bun.zig#L3177): `assertionFailure`
- [`lockfile.zig:4902`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/install/lockfile.zig#L4902): `save`
- [`lockfile.zig:1643`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/install/lockfile.zig#L1643): `saveToDisk`
- [`install.zig:10942`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/install/install.zig#L10942): `installWithManager`
- [`filter_run.zig:428`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/cli/filter_run.zig#L428): `install`
- [`cli.zig:1440`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/cli.zig#L1440): `start`
- [`cli.zig:62`](https://github.com/oven-sh/bun/blob/89d25807fbf4488d3bfcb116484d0c2cf824f98c/src/cli.zig#L62): `start`

**Comment by stefanp2021** [comment]
I removed the lock file and it works, thanks!

**Comment by dylan-conway** [comment]
fixed with #11051
